client/src/RatingsAndReviews/Reviews/IndividualReviewTile.jsx

```diff
@@ -75,9 +79,21 @@ class IndividualReviewTile extends React.Component {
             {createResponseDiv(this.props.review.response)}
             <div className="irt-helpfulness-info">
               <div className="irt-helpful-text">Helpful?</div>
-              <div className="irt-yes-clickable" onClick={this.handleHelpful}>Yes</div>
+              <div
+                interaction="helpful"
+                className="irt-yes-clickable"
+                onClick={(e) => {
+                  this.handleHelpful(e);
+                  reviewsInteraction(e);
```

Comment:
an example of the renderprops functgion that would go here would be just like you have it here - except replace reviewsInteraction function with a renderProps one =)